lib/country_select/tag_helper.rb

```diff
@@ -35,27 +35,28 @@ def country_option_tags
 
     private
     def locale
-      @options[:locale]
+      @options[:locale] || ::CountrySelect::DEFAULTS[:locale]
```

Comment:
I think getting these values should occur through an explicit `fetch` to ensure that these methods are still capable of returning the exact value they were passed to avoid any hidden behaviors. For instance, if I pass `locale: nil` I wouldn't necessarily expect the default locale to then be chosen.

It may be possible to reverse_merge `@options` with `DEFAULTS` and avoid changing all these internal helper methods, though I'm not sure where that could go right now.

Reply:
for example: `@options.fetch(:locale, ::CountrySelect::DEFAULTS[:locale])`

Reply:
👍